### PR TITLE
Cluster load: Mute Prometheus query logs for logging cluster metrics in the test background + few minor fixes in ClusterLoad

### LINKED ISSUE #2525: Cluster utilization metrics, printed if `--log-cluster-utilization` is passed, causes the execution log to be flooded
See https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10004/consoleFull

```
18:49:35 - Thread-1 - /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/utility/prometheus.py - INFO - Performing prometheus instant query '(sum(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])))' for timestamp 1594925375.386718
18:49:35 - Thread-1 - /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/utility/prometheus.py - INFO - Performing prometheus instant query 'cluster:ceph_disk_latency:join_ceph_node_disk_irate1m' for timestamp 1594925375.574387
18:49:35 - Thread-1 - /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/utility/prometheus.py - INFO - Performing prometheus instant query 'sum(rate(ceph_pool_wr[1m])) + sum(rate(ceph_pool_rd[1m]))' for timestamp 1594925375.75384
18:49:35 - Thread-1 - /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/utility/prometheus.py - INFO - Performing prometheus instant query 'ceph_cluster_total_used_bytes' for timestamp 1594925375.927908
18:49:36 - Thread-1 - ocs_ci.ocs.cluster_load - INFO - 
===============================
Cluster throughput: 96.44 MB/s
Cluster latency: 255.13 ms
Cluster IOPS: 824.45
(34.52% of the 2388.6 limit)
Cluster used space: 90.38 GB
Number of pods running FIO: 3
===============================
```

### PR DESCRIPTION
**Pull request by ebenahar**
Fixes #2525

Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Comment by ebenahar** [comment]
Verified with the last commit: deployment + tier1 of tests/manage/pv_services/ with `--io-in-bg` and `--log-cluster-utilization`

[logs.zip](https://github.com/red-hat-storage/ocs-ci/files/4943468/logs.zip)

File: ocs_ci/ocs/cluster_load.py [hunk 1]
```diff
@@ -82,7 +100,8 @@ def increase_load(self, rate=None, wait=True):
         new_args = [
             x for x in args if not x.startswith('--filesize=') and not x.startswith('--rate=')
         ]
-        new_args.append(f"--filesize={self.io_file_size}")
+        io_file_size = f"{self.pvc_size * 1000 - 200}M"
```
Comment (petr-balogh) [inline ocs_ci/ocs/cluster_load.py:103]: I am out of the context here, can you just explain to me what this size is exactly above? Thanks

File: ocs_ci/ocs/cluster_load.py [hunk 2]
```diff
@@ -274,67 +296,100 @@ def get_query(self, query):
...
+        """
+        latency = self.calc_trim_metric_mean(
+            constants.LATENCY_QUERY, mute_logs=True
+        )
+        if latency > 0.4 and len(self.dc_objs) > 0:
+            msg = (
+                f"Latency is higher than 400 ms ({latency * 1000:.2f} ms)."
+                f" Lowering IO load by deleting an FIO pod that is running"
+                f" in the test background. Once the latency drops back to "
+                f"less than 200 ms, FIO pod will be re-spawned"
```
Comment (petr-balogh) [inline ocs_ci/ocs/cluster_load.py:385]: you are not exposing any variable here so you should omit the `f` here

File: ocs_ci/ocs/cluster_load.py [hunk 3]
```diff
@@ -274,67 +296,100 @@ def get_query(self, query):
...
+            )
+            logger.warning(wrap_msg(msg))
+            self.decrease_load(wait=False)
+        if latency < 0.2 and self.target_pods_number > len(self.dc_objs):
+            msg = (
+                f"Latency is lower than 200 ms ({latency * 1000:.2f} "
+                f"ms). Re-spinning FIO pod"
```
Comment (petr-balogh) [inline ocs_ci/ocs/cluster_load.py:392]: remove `f` here

File: ocs_ci/ocs/cluster_load.py [hunk 4]
```diff
@@ -19,6 +19,22 @@
...
+    Necessary for making cluster load background logging distinguishable
+
+    Args:
+        msg (str): The log message to wrap
+
+    Returns:
+        str: The wrapped log message
+
+    """
+    marks = "=" * len(msg)
```
Comment (petr-balogh) [inline ocs_ci/ocs/cluster_load.py:34]: I think there shoule be some maximum set as if message will be to long the `=` character will will go to another line and so on, so I think about some maximum number like 120 here or even less like 80? And maybe you don't have to calculate it and just use static one for all cases?

File: ocs_ci/ocs/cluster_load.py (hunk 1 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py:103]: This is the file size used for FIO. I set it to be according to the size of the PVC which is attached to the FIO pod, minus 200MB.

File: ocs_ci/ocs/cluster_load.py (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py:385]: The first line of the message has a variable. Are you sure it is ok?

File: ocs_ci/ocs/cluster_load.py (hunk 3 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py:392]: same

File: ocs_ci/ocs/cluster_load.py (hunk 4 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py:34]: yes, good idea, will add this

File: ocs_ci/ocs/cluster_load.py (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py]: NVM, fixed

File: ocs_ci/ocs/cluster_load.py (hunk 3 above)
Comment (ebenahar) [reply ocs_ci/ocs/cluster_load.py]: fixed

**Comment by ebenahar** [comment]
validation job for the last commit is running, I saw it is logging well, so considered as verified
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10094/console

**Pull request merged**